apps/vaporgui/PColorSelector.cpp

```diff
@@ -0,0 +1,36 @@
+#include "PColorSelector.h"
+#include "QColorWidget.h"
+#include <vector>
+#include <vapor/ParamsBase.h>
+
+PColorSelector::PColorSelector(const std::string &tag, const std::string &label)
+: PLineItem(tag, _colorWidget = new QColorWidget)
```

Comment:
How is `_colorWidget` destroyed if there's no parent/child relationship specified?

Reply:
Oops, got it, sorry!